inventory/sample/group_vars/all/all.yml

```diff
@@ -112,3 +112,9 @@ no_proxy_exclude_workers: false
 
 ## Check if access_ip responds to ping. Set false if your firewall blocks ICMP.
 # ping_access_ip: true
+
+## Webhooks operating in the kube-system namespace are a receipt for disaster:
```

Comment:
Do you mean "recipe"? 
Or has one already purchased the disaster?  ;)

Reply:
If I had to pay a fine every time I confuse those two words, I would probably live under the bridge. :joy:

Thanks! Fixed!